specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.6-preview.1/certificates-sdk.json

```diff
@@ -0,0 +1,2587 @@
+          },
+          "x-ms-identifiers": []
+        },
+        "innererror": {
+          "$ref": "#/definitions/Azure.Core.Foundations.InnerError",
+          "description": "An object containing more specific information than the current object about the error."
+        }
+      },
+      "required": [
+        "code",
+        "message"
+      ]
+    },
+    "Azure.Core.Foundations.ErrorResponse": {
+      "type": "object",
+      "description": "A response containing error details.",
+      "properties": {
+        "error": {
+          "$ref": "#/definitions/Azure.Core.Foundations.Error",
+          "description": "The error object."
+        }
+      },
+      "required": [
+        "error"
+      ]
+    },
+    "Azure.Core.Foundations.InnerError": {
+      "type": "object",
+      "description": "An object containing more specific information about the error. As per Microsoft One API guidelines - https://github.com/Microsoft/api-guidelines/blob/vNext/Guidelines.md#7102-error-condition-responses.",
+      "properties": {
+        "code": {
+          "type": "string",
+          "description": "One of a server-defined set of error codes."
+        },
+        "innererror": {
+          "$ref": "#/definitions/Azure.Core.Foundations.InnerError",
+          "description": "Inner error."
+        }
+      }
+    },
+    "BackupCertificateResult": {
+      "type": "object",
+      "description": "The backup certificate result, containing the backup blob.",
+      "properties": {
+        "value": {
+          "type": "string",
+          "format": "byte",
+          "description": "The backup blob containing the backed up certificate."
+        }
+      }
+    },
+    "CertificateAttributes": {
+      "type": "object",
+      "description": "The certificate management attributes.",
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "description": "Determines whether the object is enabled."
+        },
+        "nbf": {
+          "type": "integer",
+          "format": "unixtime",
+          "description": "Not before date in UTC.",
+          "x-ms-client-name": "notBefore"
+        },
+        "expires": {
+          "type": "integer",
+          "format": "unixtime",
+          "description": "Expiry date in UTC."
+        },
+        "created": {
+          "type": "integer",
+          "format": "unixtime",
+          "description": "Creation time in UTC.",
+          "readOnly": true
+        },
+        "updated": {
+          "type": "integer",
+          "format": "unixtime",
+          "description": "Last updated time in UTC.",
+          "readOnly": true
+        },
+        "recoverableDays": {
+          "type": "integer",
+          "format": "int32",
+          "description": "softDelete data retention days. Value should be >=7 and <=90 when softDelete\nenabled, otherwise 0.",
+          "readOnly": true
+        },
+        "recoveryLevel": {
```

Comment:
This - and in all schemas - should be a string. Our SDKs do not emit this as an enum by request because it should not be compared by customers. So we should treat it as simply a string, if possible. It has changed in the past and probably will again at some point.

/cc @jlichwa @jackrichins

Reply:
Is enum -> string a breaking change?

Reply:
From a REST API perspective? Not really. The `type` is still `string`. It's extended by an enum format. Wire format is the same, and our SDKs already treat it as an opaque string.

Regardless, the service team does not want customers comparing it to anything. Even if it were considered a breaking change it's one we're likely to approve because that matches it's intent.

Reply:
In the Go SDK we use an autorest transform to change the type to `string`. So, at least for Go, there would be no breaking change.

Reply:
All the other languages already treat it like a string anyway as well.